packages/aws_bedrock/data_stream/runtime/fields/fields.yml

```
@@ -0,0 +1,55 @@
+- name: aws_bedrock
+  type: group
+  fields:
+    - name: runtime
+      type: group
+      fields:
+        - name: Invocations.sum
+          type: long
+          metric_type: gauge
+          description: The number of requests to the `Converse`, `ConverseStream`, `InvokeModel`, and `InvokeModelWithResponseStream` API operations.
+        - name: InvocationLatency.avg
+          type: long
+          unit: ms
+          metric_type: gauge
+          description: The average latency of the invocations.
+        - name: InvocationClientErrors.sum
+          type: long
+          metric_type: gauge
+          description: The number of invocations that result in client-side errors.
+        - name: InvocationServerErrors.sum
+          type: long
+          metric_type: gauge
+          description: The number of invocations that result in AWS server-side errors.
+        - name: InvocationThrottles.sum
+          type: long
+          metric_type: gauge
+          description: The number of invocations that the system throttled.
+        - name: InputTokenCount.sum
+          type: long
+          metric_type: gauge
+          description: The number of tokens of text input.
+        - name: LegacyModelInvocations.sum
+          type: long
+          metric_type: gauge
+          description: The number of requests to the legacy models.
+        - name: OutputTokenCount.sum
+          type: long
+          metric_type: gauge
+          description: The number of tokens of text output.
+        - name: OutputImageCount.sum
+          type: long
+          metric_type: gauge
+          description: The number of output images.
+- name: aws.dimensions.ModelId
+  type: keyword
+  dimension: true
+- name: aws.dimensions.ImageSize
+  type: keyword
+  dimension: true
+- name: aws.dimensions.Quality
+  type: keyword
+  dimension: true
```

This doesn't change the actual mapping, but it might be more readable. `cloudwatch` could be added as a group as well under `aws` group.

```suggestion
- name: aws
  type: group
  fields:
    - name: dimensions
      type: group
      fields:
        - name: ModelId
          type: keyword
          dimension: true
        - name: ImageSize
          type: keyword
          dimension: true
        - name: Quality
          type: keyword
          dimension: true
```

---

Updated as suggested.